CHANGELOG.md

```diff
@@ -7,9 +7,13 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased changes
+
+* Phase banner now uses design systems and adds app name support (PR #613)
```

Comment:
Could we rephrase this for consistency as
- "Update phase banner component to use GOV.UK Frontend styles"
- "Update phase banner component to support application name"